app/scripts/controllers/preferences.js

```diff
@@ -90,6 +90,7 @@ export default class PreferencesController {
         showExtensionInFullSizeView: false,
         showFiatInTestnets: false,
         showTestNetworks: false,
+        stxOptIn: null, // null means we will show the STX opt-in modal to a user if they are eligible
```

Comment:
Just for clarity and explicitness, do we typically try and avoid abbreviations so could this be `smartTransactionsOptIn` or `smartTransactionsPreferred` or `smartTransactionsEnabled`?